apps/web/src/pages/integrations/components/ConnectIntegrationForm.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useForm, Controller } from 'react-hook-form';
+import styled from '@emotion/styled';
+import { IProviderConfig } from '@notifire/shared';
+import { Button, colors, Input, Switch, Text } from '../../../design-system';
+import { Check } from '../../../design-system/icons';
+
+export function ConnectIntegrationForm({ provider }: { provider: IProviderConfig | null }) {
+  const { handleSubmit, control } = useForm();
+
+  return (
+    // eslint-disable-next-line no-console
+    <form onSubmit={handleSubmit((data) => console.log(data))}>
+      <ColumnDiv>
+        <InlineDiv>
+          <span style={{ marginRight: 5 }}>Read our guide on where to get the credentials </span>
+          <a href={provider?.docReference}>here</a>
+        </InlineDiv>
+        {provider?.credentials.map((credential) => (
+          <Controller
```

Comment:
```suggestion
          <Controller
          key={credential.key}
```

When doing a map in react it usually asks you to add key on the iterable element for it to track

Reply:
The key is added on #380 😊